main/glossary/README.md

```diff
@@ -44,42 +54,84 @@ In Agoric documentation, *contract* usually refers to a contract's source code t
 
 ## Handle
 A handle is a unique identifier implemented as a JavaScript object. Only its identity is meaningful, so handles do not have properties. Unlike number or string identifiers, handles are unforgeable. This means the only way to know a handle identity is being given an object reference, and no identity can be guessed and no fake identity will succeed. 
 
 For example, Zoe often uses `offerHandle` to refer to offers. Zoe contracts can use an offer's `offerHandle` as the key for requesting the current allocation of an offer or reallocating the offer's assets.
-
+<a name="Issuers"></a>
 ## Issuer
-Can create empty purses and payments, but cannot mint new amounts. Issuers can also transform payments (splitting, combining, burning, and claiming payments exclusively). You should get an issuer from a trusted source and then rely on it to determine if an untrusted payment is valid. Issuers are linked to a single mint and vice versa, so each issuer only works with one type of thing, such as only working with quatloos or only working with moola.
+Issuers are linked to a single [mint](#Mints) and vice versa, so each issuer works
+with one and only one asset type, such as only working with quatloos or only working
+with moola. They can create empty [purses](#Purses) and [payments](#Payments) for
+their asset type, but cannot mint new [amounts](#Amounts). 
 
-## MathHelpers
-Arithmetic on extents. MathHelpers are used by AmountMath to do their extent arithmetic, and then brand the result, making a new amount.
+Issuers can also transform
+payments of their asset type (splitting, combining, burning, and exclusively claiming
+payments). An issuer from a trusted source can determine if an untrusted payment of
+its asset type is valid. 
 
+See the [ERTP Guide's Issuer section](https://agoric.com/documentation/ertp/guide/issuer.html) for more information.
+
+<a name="MathHelpers"></a>
+## MathHelpers
+A set of methods for performing arithmetic operations on an [amount's](#Amounts) [extent](#Extents). 
+[AmountMath](#AmountMath) uses MathHelpers to do extent arithmetic, after which it brands
+the result to create a new [amount](#Amounts). For more information, see the 
+[MathHelpers ERTP Guide entry](https://agoric.com/documentation/ertp/guide/math-helpers.html).
+<a name="Mints"></a>
 ## Mint
-The admin facet of the issuer, and the only object with the authority
-to mint new digital assets.
+In ERTP, mints create digital assets and are the only objects with the authority to do so. 
+Access to a mint gives you the power to create more digital assets of its type at will. 
+Mints are [issuer's](#Issuers) admin facets, and there is a one-to-one relationship between an issuer and
+its mint.
+
+See the [ERTP Guide's Mint section](https://agoric.com/documentation/ertp/guide/mint.html) for more information.
+
+<a name="Non-fungible"></a>
+## Non-fungible
+A non-fungible asset is one where exemplars of the asset have unique individual properties and
```

Comment:
"exemplars of the asset" reads strangely to me and probably won't be well understood for anyone with English as a second language